package.json

```diff
@@ -83,11 +83,12 @@
     "eslint-plugin-react-hooks": "^4.1.2",
     "gh-pages": "^3.0.0",
     "husky": "^4.3.0",
+    "identity-obj-proxy": "^3.0.0",
     "jest": "^26.6.3",
     "lint-staged": "^10.0.7",
     "prettier": "^1.19.1",
-    "react": "^17.0.1",
-    "react-dom": "^17.0.1",
+    "react": "16.14.0",
+    "react-dom": "16.14.0",
```

Comment:
enzyme not support react v17 now, so degrade to react v16. Waiting enzyme-adapter-react-16 update toenzyme-adapter-react-17, will update this react v17 version.